content/kots-cli/remove.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2020-11-20
+linktitle: "remove"
+title: kots remove
+weight: 90120
+---
+
+Remove application reference from Admin Console.
+Multiple applications can be installed using the `Add a new application` button in Admin Console or using the `kots install` command.
+All installed applications will appear in Admin Console.
+The `kots remove` command can be used to remove them.
+Note that deployed application will not be removed from the cluster, only it's reference will be removed from Admin Console.
+
+### Usage
+```bash
+kubectl kots remove [app-slug]
+```
+* _Replace `[app-slug]` with the slug for the KOTS application that should be removed (required)._
+* _If the KOTS application has been packaged by Replicated Vendor, the `--license-file` flag must be provided._
+* _Provide `[flags]` according to the table below_
+
+This command supports all [global flags](/kots-cli/global-flags/) and also:
+
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `--force` |  bool  |  if application was deployed, the command will fail unless the force flag is specified |
+
+### Example
+```bash
+kubectl kots remove sentry -n default
```

Comment:
The text above says a license is required for nearly all instances, so maybe it should be provided here? (or drop that requirement)

Reply:
that was just copy/paste stuff from another doc